threedi_modelchecker/tests/test_checks_raster.py

```diff
@@ -238,6 +239,20 @@ def test_has_projection_err(tmp_path, interface_cls):
     assert not check.is_valid(path, interface_cls)
 
 
+@pytest.mark.parametrize(
+    "interface_cls", [GDALRasterInterface, RasterIORasterInterface]
+)
+@pytest.mark.parametrize(
+    "raster_epsg, sqlite_epsg, validity",
+    [(28992, 28992, True), (28992, 1, False), (None, None, True), (None, 1, False)],
+)
+def test_has_epsg(tmp_path, interface_cls, raster_epsg, sqlite_epsg, validity):
+    path = create_geotiff(tmp_path / "raster.tiff", epsg=raster_epsg)
+    check = RasterHasMatchingEPSGCheck(column=models.GlobalSetting.dem_file)
+    check.epsg_code = sqlite_epsg
+    assert check.is_valid(path, interface_cls) == validity
+
+
```

Comment:
Kan je `test_raster_check_invalid_file` aanvullen met de nieuwe check die je hebt gemaakt?

Reply:
Dat gaat helaas mis doordat the check instance een `self.epsg_code` nodig heeft om te werken. In `test_has_epsg` wordt deze handmatig gedefiniëerd (`check.epsg_code = sqlite_epsg`), waardoor het goed gaat. In `test_raster_check_invalid_file` is er geen eenvoudige optie om voor alleen een check een attribute te definiëren.

De test kan worden herschreven door
- een `if` voor de epsg code definitie in de test toe te voegen
- te parametriseren

maar het is de vraag of dat gewenst is.